assets/queries/cloudFormation/iam_password_without_lowercase_letter/query.rego

```diff
@@ -4,6 +4,7 @@ CxPolicy[result] {
 	resource := input.document[i].Resources[name]
 	password := resource.Properties.LoginProfile.Password
 	not regex.match(".*[a-z]", password)
+	is_string(password)
```

Comment:
move this check to the line above, before the regex match, to stop the execution in a earlier stage if it's not string